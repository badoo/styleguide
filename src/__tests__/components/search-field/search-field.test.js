import React from 'react';
import renderer from 'react-test-renderer';
import SearchField from '../../../components/search-field/search-field';
import 'jest-styled-components';

describe('SearchField generic tests', () => {
    let component = renderer.create(
        <SearchField value={'comp'} onChange={(event) => `we test output of ${event}`} />
    );

    it('When we search', () => {
        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});
