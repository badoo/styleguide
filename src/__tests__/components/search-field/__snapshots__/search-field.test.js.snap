// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchField generic tests When we search 1`] = `
<div
  className="eTRikO"
>
  <input
    autoComplete="off"
    className="fIzlLk"
    onChange={[Function]}
    placeholder="Search component name"
    type="text"
    value="comp"
  />
  <div
    className="kZvQyL"
  >
    <div
      className="bHoZMY"
      role="presentation"
      size={0}
    >
      <svg
        viewBox="0 0 32 32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M20.5 20.92l7.62 7.67-1.43 1.4-7.72-7.79A10.83 10.83 0 0 1 2 13.28v-.45a10.84 10.84 0 0 1 21.67 0v.45c0 2.98-1.21 5.68-3.16 7.64zM4 12.82v.46a8.83 8.83 0 0 0 17.67 0v-.45a8.83 8.83 0 0 0-17.67 0z"
          fillRule="evenodd"
        />
      </svg>
    </div>
  </div>
</div>
`;
